pymemcache/serde.py

```diff
@@ -105,3 +105,53 @@ def python_memcache_deserializer(key, value, flags):
             return None
 
     return value
+
+
+class PythonMemcachePickleSerde(object):
```

Comment:
It seems like we should name this just `PickleSerde`. What do you think?

```suggestion
class PickleSerde(object):
```

Reply:
I'm a huge 👍 on the more concise name. I think I was just trying to stay consistent with the things which are already in this module.